metabrainz/templates/index/projects.html

```diff
@@ -0,0 +1,134 @@
+{% extends 'base.html' %}
+
+{% block title %}Projects - MetaBrainz Foundation{% endblock %}
+
+{% block content %}
+<h1 class="page-title">The MetaBrainz Foundation Projects</h1>
+<div id="projects">
+  
+  <div class="media">
+    <div class="media-left media-middle">
+      <a href="https://musicbrainz.org/">
+        <img class="media-object logo" src="{{ url_for('static', filename='img/projects/MBnt.png') }}" alt="MusicBrainz logo">
```

Comment:
I think it'd be better to follow the naming scheme from the metabrainz-logos repository (I think that's "ProjectName_no_text.ext"?).

We'll eventually want to do a ["StaticBrainz"](http://tickets.musicbrainz.org/browse/MBH-413) site for holding all our static resources (like images), so the more identically they're named across instances where they're used, the easier it will be to find them and change them to point to staticbrainz.org instead of their own site. :)